core/src/main/java/org/apache/carbondata/core/scan/filter/executer/RangeValueFilterExecuterImpl.java

```diff
@@ -89,7 +90,10 @@ public RangeValueFilterExecuterImpl(DimColumnResolvedFilterInfo dimColEvaluatorI
     isRangeFullyCoverBlock = false;
     initDimensionChunkIndexes();
     ifDefaultValueMatchesFilter();
-
+    if (isDimensionPresentInCurrentBlock == true) {
```

Comment:
Just use ` if (isDimensionPresentInCurrentBlock)`

Reply:
Ok i will change, I haven't change this code may be because of formatting:)